requests/rustac-add-feedstock-output.yml

```diff
@@ -0,0 +1,3 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  - rustac: rustac
```

Comment:
@gadomski, just to double check, you want the package name for rustac's Python bindings to be just `rustac` and not `rustac-py`? Do you want to have R bindings `rustac-r` as mentioned at https://github.com/stac-utils/rustac/issues/641 in the future? And potentially pure Rust `rustac` as well?

Reply:
>  you want the package name for rustac's Python bindings on conda-forge to be just rustac and not rustac-py

Yeah, I think this is correct ... I'm leaning in to letting Python and Rust share the **rustac** name. Since https://github.com/stac-utils/rustac-py includes a wrapper around the Rust CLI, we shouldn't need the Rust package on Conda. If we make an R one, we'll have to use **rustac-r** 🤷🏼.

Reply:
It will be `r-rustac`, we prefix all R packages on conda-forge.

Reply:
Ok, then should be all good to go :rocket: